Move NamespaceATM.cs to Lesson 6/[Sheuchyk]

alenasheuchyk opened the pull request:
Hello Alex, here is my homework for ATM, sorry for being late

Afoodzya commented on Lesson 6/[Sheuchyk]/NamespaceATM.cs:
```diff
@@ -0,0 +1,134 @@
+        }
+        public void ShowBalance()
+        {
+            Console.WriteLine($"Current balance: {balance}  BEL RUB");
```
use either BYN or RUB as you do on 38 and 46 lines to be consistent in abbreviation

Afoodzya commented on Lesson 6/[Sheuchyk]/NamespaceATM.cs:
```diff
@@ -0,0 +1,134 @@
+            }
+
+            balance -= amount;
+            string record = $"{DateTime.Now}: Withdrawal -{amount} RUB";
```
Raw DateTime.Now is hard to read:
``string record = $"{DateTime.Now:yyyy-MM-dd HH:mm:ss}: Deposit +{amount} BYN";``

alenasheuchyk commented:
Hi Alex, I’ve updated the currency abbreviation to use a consistent format (BYN across all methods). Also adjusted the DateTime.Now format to yyyy-MM-dd HH:mm:ss as you suggested for better readability. Thank you for your help! :)
